spring-cloud-commons/src/test/java/org/springframework/cloud/client/discovery/noop/NoopDiscoveryClientConfigurationAdditionalTests.java

```diff
@@ -22,6 +23,7 @@
  */
 @RunWith(SpringRunner.class)
 @SpringBootTest(classes = NoopDiscoveryClientConfigurationAdditionalTests.App.class)
+@Ignore
```

Comment:
These were tests related to the DiscoveryClient defaulting to the NoOp implementation, with SimpleDiscoveryClient the defaults have changed. I have now removed this test and have another test where the SimpleDiscoveryClient as default is validated.